jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/main/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DeploymentDescriptorViewImpl.ui.xml

```diff
@@ -216,6 +216,17 @@
                             </b:Column>
                         </b:FormGroup>
 
+                        <!-- limit serialization classes -->
+                        <b:FormGroup ui:field="limitSerializationClassesGroup">
+                            <b:FormLabel for="limitSerializationClasses" ui:field="limitSerializationClassesLabel" addStyleNames="col-md-3">
+                                <ui:text from="{i18n.LimitSerializationClasses}"/>
```

Comment:
looks like this i18n should point to org.jbpm.console.ng.wi.client.i18n.Constants instead of org.jbpm.console.ng.bd.client.i18n.Constants that is currently defined in this file. Looks like a left over from move of deployment descriptor to workbench integration module though without this change console does not build - its showcase module